src/Form/FormAnalyzer.js

```diff
@@ -277,7 +290,8 @@ class FormAnalyzer {
             this.updateSignal({ string, strength, signalType: `external link: ${string}`, shouldFlip });
         } else {
             // any other case
-            this.updateSignal({ string, strength: 1, signalType: `generic: ${string}`, shouldCheckUnifiedForm: true });
+            const isH1Element = el.tagName === 'H1';
+            this.updateSignal({ string, strength: isH1Element ? 3 : 1, signalType: `generic: ${string}`, shouldCheckUnifiedForm: true });
         }
```

Comment:
As discussed on MM (from your feedback), "Sign up" text is poorly scored, even though it's the largest element (h1) in the container.